src/core/reopt_inputs.jl

```diff
@@ -215,18 +239,26 @@ end
 Create data arrays associated with techs necessary to build the JuMP model.
 """
 function setup_tech_inputs(s::AbstractScenario)
+    #TODO: new boiler, steam turbine, ghp
+    #TODO: create om_cost_per_kwh in here as well as om_cost_per_kw?
```

Comment:
@hdunham  should we add this as an issue?

Reply:
issue created